docs/migration-guides/1.10-1.11.md

```diff
@@ -131,4 +141,58 @@ Perform the following steps to see if the migration process was successful.
 
 After the migration, use the new Knative Eventing Mesh endpoint, `https://gateway.domain/APP_NAME/events`, to send events.
 
-**NOTE**: The old endpoint, `https://gateway.domain/APP_NAME/v1/events`, will be handled by the compatibility layer which uses the new Eventing Mesh to route events from the old endpoints to the sinks.
\ No newline at end of file
+**NOTE**: The old endpoint, `https://gateway.domain/APP_NAME/v1/events`, will be handled by the compatibility layer which uses the new Eventing Mesh to route events from the old endpoints to the sinks.
```

Comment:
```suggestion
> **NOTE:** The deprecated `https://gateway.domain/APP_NAME/v1/events` endpoint will be handled by the compatibility layer which uses the new Eventing Mesh to route events from the deprecated endpoints to the sinks.
```

Reply:
`deprecated endpoints` - so is there one deprecated endpoint or more?

Reply:
No should be this one - I  changed this line a bit to also remove the duplications.